README.md

```diff
@@ -154,6 +154,12 @@ public void some_important_test() throws Exception {
 }
 ```
 
+## Magic that Barista does for you
+
+In order to speed up testing, Barista keeps in mind some considerations.
+- **Scrolls when needed**: Filling a form inside a `ScrollView` requests you to scroll to the next item several times to avoid interacting with undisplayed widgets. To keep tests concise, Barista scrolls automatically before clicking a `Button` and before writing into an `EditText`.
```

Comment:
Oh! Both were wrong. As you mentioned, all methods do the magic, in exception of `BaristaRadioButtonActions`. Opening an issue called #54!